<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>296</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="slider_Quality">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LoQ Headset</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_A2DP">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_HFP">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gain In</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HiQ Audio</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="device_A2DP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="device_HFP">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPairing">
    <property name="title">
     <string>Pairing</string>
    </property>
    <addaction name="actionDelete_All"/>
    <addaction name="actionScan"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSerial">
    <property name="title">
     <string>Serial</string>
    </property>
   </widget>
   <addaction name="menuSerial"/>
   <addaction name="menuPairing"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDelete_All">
   <property name="text">
    <string>Delete All</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="text">
    <string>Scan</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
